components/net_traits/request.rs

```diff
@@ -254,6 +254,8 @@ pub struct RequestBuilder {
     pub initiator: Initiator,
     pub https_state: HttpsState,
     pub response_tainting: ResponseTainting,
+    /// Servo internal: if crash details are present, trigger a crash error page with these details
```

Comment:
Ditto.

Reply:
fixed in c85c102311a8914f8a34f0fb6932be6b2f163382